test/Scientist.Test/ThrownTests.cs

```diff
@@ -0,0 +1,136 @@
+﻿using GitHub;
+using NSubstitute;
+using System;
+using UnitTests;
+using Xunit;
+
+/// <summary>
+/// Tests all of the conditions that 
+/// </summary>
+public class ThrownTests
+{
+    [Fact]
+    public void CompareOperation()
+    {
+        const int expectedResult = 42;
+
+        var mock = Substitute.For<IControlCandidate<int>>();
+        mock.Control().Returns(expectedResult);
+        mock.Candidate().Returns(0);
+
+        var ex = new Exception();
+
+        var result = Scientist.Science<int>(nameof(CompareOperation), experiment =>
+        {
+            experiment.Thrown(mock.Thrown);
+            experiment.Compare((x, y) =>
+            {
+                throw ex;
+            });
+            experiment.Use(mock.Control);
+            experiment.Try(mock.Candidate);
+        });
+
+        Assert.Equal(expectedResult, result);
+        mock.Thrown(Operation.Compare, ex);
```

Comment:
Did you mean to verify that the `Thrown` method was called with these parameters on the mock?